admin_guide/manage_authorization_policy.adoc

```diff
@@ -614,10 +615,31 @@ $ oc project <project_you_want_to_add_the_local_role_exampleview_to>
 
 Optionally, annotate it with a description.
 
-To use the new role, run:
+To create policybinding.yaml:
```

Comment:
I'd say something like, "Save the following role binding in the *_policybinding.yaml_* file:"

I might also add the `cat` command to save the file.